Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverOnboarding.yaml

```
@@ -1340,6 +1342,7 @@ FleetOwnerDocumentVerificationConfig:
     isApprovalSupported: Maybe Bool
     onlyImageVerificationStatusLookupRequired: Maybe Bool
     documentFields: Maybe [Domain.Types.DocumentVerificationConfig.FieldInfo]
+    isApprovalSupported: Maybe Bool
```

YAMLlint (1.37.1)

[error] duplication of key "isApprovalSupported" in mapping

(key-duplicates)